src/compas_rhino/conversions/meshes.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from itertools import izip_longest  # type: ignore
+
+from System.Drawing import Color as SystemColor  # type: ignore
+from System.Array import CreateInstance  # type: ignore
+from Rhino.Geometry import Mesh as RhinoMesh  # type: ignore
```

Comment:
since we have our own `RhinoMesh` class, wouldn't it be clear to `import Rhino.Geometry as rg` and then reference as `rg.Mesh`?

Reply:
`RhinoMesh` has been removed. everything is now more transparently in the conversion functions...